packages/api/src/elastic/pages.ts

```diff
@@ -789,6 +790,8 @@ export const updatePages = async (
       return null
     }
 
+    // TODO: publish entityUpdated events for each page
```

Comment:
oh good TODO.

Reply:
haha, need to wrap up a new way to publish bulk events